### ISSUE TITLE
base plugin problem with filefactory

### ISSUE THREAD

**Issue by erbses** [pyload/pyload#2300]
Hello,
i use the pyload on my NAS (Synology).
I installed the Pyload.spk from "unzurechnende.info".
Then i changed the 
accounts.py in /volume1/@appstore/pyload/pyload/module/plugins/accounts and now, the premium acount on filefactory is verified. BUT:
i can`t download anything because there is a problem in the base plugin.
(  **version** = "0.14")
i just download a 5.3kiB big file. 
"Fertig 5.30 KiBBasePlugin"

when i change the plugin to an other version (not the one which is direcly installed) i get the following message:

'NoneType' object has no attribute 'BasePlugin' 0.00 BBasePlugin

In the downloaded file (5.3kb) is some text in a html file (written).
There is a downloadlink embeddeb, which works, but only when i read the file and copy the link to my pyload on my NAS as a free download. It not supports the premium account speed.

I hope you understand.

**Comment by GammaC0de** [comment]
Sorry, don't understand your problem

**Comment by sraedler** [comment]
which Hoster?

do you add the page including the link of The Hoster or do you add jetzt Hoster Link?

**Comment by sraedler** [comment]
so do you add the link of the page including the filefactory link or do you add the filefactory link itself?

which version has the filefactory plugin?

**Comment by erbses** [comment]
I search via Filefactory a file.
Then i Click one result and After that i copy the Link from the Browser.
But i already tried to use the "right Mouse Button" on the Field "Download with Premium Account" and copied that Link to Pyload too .
Best Regards,

Michael

**Comment by erbses** [comment]
here are the facts: ;)

FilefactoryCom(Account):
    __name__ = "FilefactoryCom"
    __version__ = "0.14"
------------------------------------------------------------------------
class FilefactoryCom(SimpleHoster):
    __name__    = "FilefactoryCom"
    __type__    = "hoster"
    __version__ = "0.61"
------------------------------------------------------------------------
class BasePlugin(Hoster):
    __name__ = "BasePlugin"
    __type__ = "hoster"
    __version__ = "0.14"

what do you need more?
thanks for your help.

You have a legal file for me to try to download from filefactory (compare between free and premium)?
let me know,
Michae

**Comment by GammaC0de** [comment]
Can this be closed?

**Comment by erbses** [comment]
No, i don't think so.
I found out something else:

When I am at home, searching and copiing a link to my NAS which is also at home, works everything fine. Same wifi.

When I search a link in the office or on my mobilephone via lte, umts or in another wifi-network and copy the link to my NAS at home, the links are empty (or 5.6 kb small).

So it looks like Filefactory is controlling the links, if they come from the same IP(?) adress like the downloading IP.

That is something I recognized.

Perhaps somebody van verify that....

Best Regards,

Michael
